Config/Inc/config.h

```diff
@@ -31,16 +34,19 @@ typedef enum {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE = 3, OPENWIRE
 
 //--------------------------------------------------------------------------------
 // MAX and MIN limits of lithium ion cells
-#define MIN_VOLTAGE_LIMIT				2.7		// Under voltage limit (Voltage)	(actual min: 2.5V)
-#define MAX_VOLTAGE_LIMIT				4.0		// Over voltage limit (Voltage)		(actual max: 4.2V)
+#define MIN_VOLTAGE_CHARGING_LIMIT      2.55F     // Under voltage limit for charging
+#define MIN_VOLTAGE_LIMIT				2.7F		// Under voltage limit (Voltage)	(actual min: 2.5V)
+#define MAX_VOLTAGE_LIMIT				4.0F		// Over voltage limit (Voltage)		(actual max: 4.2V)
 
-#define MAX_DISCHARGE_TEMPERATURE_LIMIT	73.00	// Max temperature limit (Celcius)	(actual max: 75C)
-#define MAX_CHARGE_TEMPERATURE_LIMIT	48.00	// Max temperature limit (Celcius)	(actual max: 50C)
+#define MAX_DISCHARGE_TEMPERATURE_LIMIT	60.00F	// Max temperature limit (Celcius)	(recommended release: 60.00C)
+#define MAX_CHARGE_TEMPERATURE_LIMIT	45.00F	// Max temperature limit (Celcius)	(recommended release: 45.00C)
 
-#define MAX_CURRENT_LIMIT				100000		// Max current limit (Milliamperes)		(Max continuous discharge is 15A per cell)
+#define MAX_CURRENT_LIMIT				75000		// Max current limit (Milliamperes)		(Max continuous discharge is 15A per cell)
 #define MAX_HIGH_PRECISION_CURRENT 		50000		// Max current detectable by the high-precision current sensor (mA)
```

Comment:
Should we get rid of this since we are using a different current sensor now?

Reply:
If it doesn't make sense anymore then yeah. Just make sure to alter wherever it's referenced accordingly if those places need to rely on a different value.